nimare/base/estimators.py

```diff
@@ -24,24 +23,44 @@ def __init__(self):
     def transform(self, dataset):
         """Add stuff to transformer.
         """
-        if not isinstance(dataset, Dataset):
+        if not hasattr(dataset, 'slice'):
```

Comment:
Replacing `isinstance` with `hasattr` so I can use fake datasets in tests. If there's a more elegant way to deal with it, I'd be happy to do that instead.

Reply:
Mocking might be the way to go if we anticipate this kind of thing being an issue routinely, but is probably more trouble than it's worth at this point.